generic/simple.py

```diff
@@ -16,14 +16,17 @@
 with open(f"../{device}.pickle", 'rb') as f:
     db = pickle.load(f)
 
+
+added_wires = []
```

Comment:
Just as a random unsolicited advice, using a set() would probably be faster ...

Reply:
Thanks, indeed it boost performance a lot. Didn't know this was such hot code :).